core/src/impl/Kokkos_Core.cpp

```diff
@@ -690,6 +690,9 @@ void pre_initialize_internal(const Kokkos::InitializationSettings& settings) {
 #elif defined(KOKKOS_ARCH_AMD_ZEN3)
   declare_configuration_metadata("architecture", "CPU architecture",
                                  "AMD_ZEN3");
+#elif defined(KOKKOS_ARCH_RISCV_MILKV)
+  declare_configuration_metadata("architecture", "CPU architecture", 
+                                 "Milk-V (RISC-V)")
```

Comment:
Similar comment - replace MILKV w/ something specific to the CPU arch

Reply:
I moved to SG2042.